src/main/java/com/ghilly/web/ErrorHandler.java

```diff
@@ -18,8 +18,8 @@ public ResponseEntity<String> catchRuntimeException(RuntimeException exception)
                 .body("Runtime exception " + exception.getMessage());
     }
 
-    @ExceptionHandler(IdIsNotFoundException.class)
-    public ResponseEntity<String> catchIdIsNotFoundException(IdIsNotFoundException exception) {
+    @ExceptionHandler(IdNotFoundException.class)
+    public ResponseEntity<String> catchIdIsNotFoundException(IdNotFoundException exception) {
```

Comment:
please rename to catchIdNotFoundException

Reply:
Renamed